Build.ps1

```diff
@@ -5,7 +5,7 @@
 param(
```

Comment:
Please run reformat (SHIFT+ALT+F) on the file so the formatting rules in .vscode\settings.json is applied.